examples/linkify-example/src/Link.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Plugin } from '@djsp/core'
+import linkStrategy from './linkStrategy'
+import linkifyIt from 'linkify-it'
+import tlds from 'tlds'
+import type { DraftDecoratorComponentProps } from 'draft-js'
+
+const linkify = linkifyIt()
+linkify.tlds(tlds)
+
+type Props = DraftDecoratorComponentProps & {
+  target?: string,
+  rel?: string,
+}
+
+class LinkComponent extends Component<Props> {
+  render() {
+    const {
+      decoratedText, // eslint-disable-line no-unused-vars
+      target = '_self',
+      rel = 'noreferrer noopener',
+      dir, // eslint-disable-line no-unused-vars
+      entityKey, // eslint-disable-line no-unused-vars
+      offsetKey, // eslint-disable-line no-unused-vars
+      contentState, // eslint-disable-line no-unused-vars
+      ...otherProps
+    } = this.props // eslint-disable-line no-use-before-define
```

Comment:
can we clean this up a little. we don't need to destructure all the props above, simply pass them on to the component